test/helpers/pkgMatchers.js

```diff
@@ -0,0 +1,50 @@
+import semver from "semver";
+
+const matchDependency = dependecyType => {
```

Comment:
should be `dependencyType`. I go cross-eyed typing "dependency" a lot 😭

Reply:
Addressed via https://github.com/lerna/lerna/pull/1069/commits/e4c9f0c6affcab5d07e1e3f78fb27c9a99b52268